### ISSUE TITLE
Localizable_EN.strings has issues

### ISSUE THREAD

**Issue by lmmenge** [onfido/onfido-ios-sdk#200]
### What was the expected behaviour?

`Localizable_EN.strings` should be well-formed.

### What happened instead?

- There's a localization without key:
```
"Photo of your document. Double tap to enlarge.";
```

- The following key is present twice in the file:
```
"onfido_doc_confirmation_button_secondary" = "Retake photo";
```

### Version info:

- Cocoapods (if applicable): Not applicable
- Onfido SDK (Debug/Release): 18.10.1
- iOS: 14.2
- Xcode: 12.2
- Device/Simulator: Not applicable
- Device/Simulator language: Not applicable

### Integration configuration:

Not relevant.

### Steps to reproduce:

Run a linter on the `.strings` file.

**Comment by keremgund** [comment]
Hello @lmmenge ,
Thanks for reporting the issue. We've created internal ticket on our side to fix the problem. 
I will let you know once we're about the release the fix for this problem.

Sorry for the inconvenience,
Kerem

**Comment by keremgund** [comment]
Hello @lmmenge ,
I just wanted to let you know that the fix for this problem will be delivered in the next release.
I will inform you was next release done.

Thanks,
Kerem

**Comment by anuragajwani** [comment]
@lmmenge

we have fixed the issue in 20.0.0 release. Please let us if you still have issues.

Anurag
